mappings/net/minecraft/client/option/GameOptions.mapping

```diff
@@ -45,50 +75,78 @@ CLASS net/minecraft/class_315 net/minecraft/client/option/GameOptions
 	FIELD field_1865 invertYMouse Z
 	FIELD field_1866 debugEnabled Z
 	FIELD field_1867 keySprint Lnet/minecraft/class_304;
+		COMMENT A key binding for sprinting.
+		COMMENT Bound to {@linkplain org.lwjgl.glfw.GLFW#GLFW_KEY_LEFT_CONTROL the left control key} by default.
 	FIELD field_1868 guiScale I
 	FIELD field_1869 keyDrop Lnet/minecraft/class_304;
+		COMMENT A key binding for dropping the selected item.
```

Comment:
I think this one works similarly for inventory container screens as the f swap key. Does it?